apps/wing/src/cli.ts

```diff
@@ -151,7 +158,7 @@ async function main() {
     .description(
       "Compiles a Wing program and runs all functions with the word 'test' or start with 'test:' in their resource identifiers"
     )
-    .argument("<entrypoint...>", "all entrypoints to test")
+    .argument("[entrypoint...]", "all entrypoints to test", "DEFAULT_TEST_ENTRYPOINTS")
```

Comment:
What's this?

Reply:
If you call `wing test` without passing any entrypoint, it would use `DEFAULT_TEST_ENTRYPOINTS` as the default entrypoint, which then gets resolved here: https://github.com/winglang/wing/pull/4257/files#diff-c5ea03c3011f3998e7d6a7bb7ff4a3a29f2035a6006edc2e64c9651afae78b8eR24-R29

This solution was needed in order to throw an error and point the user towards the right direction, otherwise he could be confused seeing that the command works and executes correctly despite testing 0 files.

A different approach could be to put all the logic inside a function if you don't like the approach of using a placeholder string there.

Reply:
Btw, I'm aware of the contribution guidelines. I opened this PR without updating the existing tests and docs just to see if you agreed on having this as a  feature.